types/react-native/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for react-native 0.55
+// Type definitions for react-native 0.56
```

Comment:
0.56 is still in RC and not released yet. I think we should keep the version number.